src/server/scripts/Northrend/CrusadersColiseum/TrialOfTheCrusader/boss_northrend_beasts.cpp

```diff
@@ -356,44 +351,38 @@ class npc_snobold_vassal : public CreatureScript
                     case ACTION_DISABLE_FIRE_BOMB:
                         _events.CancelEvent(EVENT_FIRE_BOMB);
                         break;
+                    case ACTION_JUMP_ON_PLAYER:
+                        if (Unit* target = SelectTarget(SELECT_TARGET_RANDOM, 0, 0.0f, true, -SPELL_SNOBOLLED))
+                        {
+                            me->RemoveFlag(UNIT_FIELD_FLAGS, UNIT_FLAG_NON_ATTACKABLE | UNIT_FLAG_NOT_SELECTABLE);
+                            _targetGUID = target->GetGUID();
+                            target->CreateVehicleKit(444, 0);
+                            me->EnterVehicle(target, 0);
+                            //DoCast(victim, SPELL_SNOBOLLED, true); direct cast dont work
```

Comment:
didnt test that part of the cast tbh, but I think its something related to target check